packages/docs/src/routes/docs/(qwikcity)/advanced/static-assets/index.mdx

```diff
@@ -8,15 +8,45 @@ contributors:
   - the-r3aper7
 ---
 
+---
+
 # Static Assets
 
-Use `/public` folder to store all static assets such as images, fonts, etc.
+The `/public` folder is utilized to store all static assets such as images, fonts, etc.
 
-All static assets are served from your server's root (`/` as in `https://example.com/`) folder and superimposed under your routes (routes have priority).
+All static assets are served from your server's root folder (`/` as in `https://example.com/`), and are superimposed under your routes. Note that routes take precedence.
 
-```bash
+```
 public/
 ├── favicon.ico           # https://example.com/favicon.ico
 └── images/
     └── logo.png          # https://example.com/images/logo.png
 ```
+
+### Accessing Static Assets within the Application
+
+To access and use static assets within your application, you can use the `useLocation` and `useResource$` functions. Below is an example that fetches a JSON file from the public folder:
+
+```jsx
+export default component$(() => {
+  const loc = useLocation();
+  const list = useResource$(() => {
+    const url = `${loc.url.origin}/myJson.json`;
```

Comment:
I don't think this is good advice. It will cause HTTP fetch on component render. I think one should just get the file directly by importing it with `import FILE from './my-file.json?inline';`